source/docs/xrp-robot/hardware-and-imaging.rst

```diff
@@ -3,7 +3,7 @@
 To get started with the XRP, you will need to have the necessary hardware.
 
 1. XRP Kit [from SparkFun](https://www.sparkfun.com/products/22230) or [from DigiKey](https://www.digikey.com/en/products/detail/sparkfun-electronics/KIT-22296/19117674) - Available at a discount for educational institutions or FIRST teams. See individual vendors for details.
-2. [Micro-USB cable](https://www.amazon.com/dp/B0711PVX6Z) - Ensure that this is a data cable
+2. [USB-C cable](https://www.amazon.com/dp/B01GGKYN0A) - Ensure that this is a data cable.  Beta versions of XRP use a [Micro-USB cable](https://www.amazon.com/dp/B07232M876)
```

Comment:
I don't think charge only USB-C cables work, so I'd move the data cable note to after the Micro-USB cable link